xcodeProject/compile_commands.json

```diff
@@ -1,3 +1,302 @@
+  "file": "/Users/robhudson/Development/xobjective-c/exercises/anagram/AnagramTest.m"
```

Comment:
I don't how oclint works here but I think these are generated automaticly.  I am not sure about checking in their contents, maybe we should exclude it in .gitignore